vars/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+ag_jenkins:
+  route:
+    name: 'jenkins'
+  service:
+    name: 'jenkins'
+    jnlp: 'jenkins-jnlp'
+  deployment:
+    name: 'jenkins'
+  pv_claim:
+    name: 'jenkins'
+
+ag_sdk:
```

Comment:
Do you think we need to use `ag_android_sdk` instead of `ag_sdk`? We will have to add other platforms in a not so distant future (ios, cordova, xamarin, etc)